draft-ietf-quic-transport.md

```diff
@@ -3088,11 +3088,11 @@ actually lost.
 
 
 The PADDING frame provides a useful option for PMTU probe packets that does not
```

Comment:
nit: Other transports don't have padding?

Reply:
Yeah, I'd drop the does not exist bit.  It's a big claim and not really necessary.